### PR TITLE
SDV2-1257 Add jenkinsfile for parallel building of images.

### PR DESCRIPTION
**Pull request by carinadigital**
Please check the current and latest tags produced

### CODE REVIEW

**Comment by Azulinho** [comment]
nope.
this is groovy, seriously, keep it DRY, use a loop

**Comment by Azulinho** [comment]
and be aware that by running it this you are not consuming the local docker cache layers and you consume 10 executors accross possibly 10 different slaves. while it maybe be tempting to use a parallel block everywhere because it looks pretty most of the time it is not the most efficient way to block resources across a distributed cluster.

**Comment by carinadigital** [comment]
I'm happy to add more logic and looping, if everyone thinks it's valuable.

**Comment by carinadigital** [comment]
The parallel task is limited to a single executor on a single slave here.

**Comment by brettminnie** [comment]
Tags look good, there is a moan in the logs about the use of `--force` in docker tag, may be in the Makefile, if so we need to remove it.

I agree on using the loop here, other than that, nice speed increase

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    // Only push a latest image if we are processing the master branch.
+                    echo "On master branch. Pushing latest image for " + inputString
+                    img.push()
+                }
+                img.push(env.NEWTAG)
+            }
+        }
+    }
+}
+
```
Comment (brettminnie) [inline Jenkinsfile]: Random whitespacing, you may need to use enter several more times here :)

**Pull request merged**